components/sysutils/nmap/Makefile

```diff
@@ -54,29 +52,33 @@ COMPONENT_PREP_ACTION = (cd $(@D)/libdnet-stripped && autoconf)
 
 COMPONENT_PRE_CONFIGURE_ACTION = $(CP) -rp $(SOURCE_DIR)/. $(@D)
 
-CONFIGURE_OPTIONS  +=   CPPFLAGS="$(CPPFLAGS)"
+COMPONENT_POST_INSTALL_ACTION = $(GSED) \
+    -i -e 's:\#\!/usr/bin/python3:\#\!/usr/bin/python$(PYTHON_VERSION):' $(PROTOUSRBINDIR)/ndiff ; \
```

Comment:
Please try to use `PYTHON_SCRIPTS` instead of this.

Reply:
removed